tools/myst/host/exec.c

```diff
@@ -129,14 +129,62 @@ long myst_wake_wait_ocall(
     return myst_tcall_wake_wait(waiter_event, self_event, ts);
 }
 
+int exec_create_enclave_setting(
+    oe_enclave_setting_t** settings,
+    uint32_t* setting_count,
+    uint8_t* root_hash,
+    size_t root_hash_size,
+    const char* config_buffer,
+    size_t config_size)
+{
+    int ret = 0;
+    oe_sgx_enclave_setting_config_data* config_data_setting_optional = NULL;
+    oe_enclave_setting_t* optional_settings = NULL;
+
+    if ((config_data_setting_optional =
+             malloc(sizeof(oe_sgx_enclave_setting_config_data))) == NULL)
+        ERAISE(-ENOMEM);
+
```

Comment:
Set `config_data_setting_optional->config_svn` to 0 if it's not used.
Also, to support both icelake and coffeelake, set `config_data_setting_optional->ignore_if_unsupported` to true.